app/assets/javascripts/discourse/app/widgets/topic-admin-menu.js

```diff
@@ -76,33 +76,44 @@ createWidget("topic-admin-menu-button", {
 
   showAdminMenu(e) {
     this.state.expanded = true;
-    let $button;
+    let button;
 
     if (e === undefined) {
-      $button = $(".keyboard-target-admin-menu");
+      button = document.getElementsByClassName(".keyboard-target-admin-menu");
     } else {
-      $button = $(e.target.closest("button"));
+      button = e.target.closest("button");
     }
 
-    const position = $button.position(),
-      SPACING = 3,
-      MENU_WIDTH = 217;
+    const position = { top: button.offsetTop, left: button.offsetLeft };
+    const spacing = 3;
+    const menuWidth = 212;
 
-    const rtl = $("html").hasClass("rtl");
-    position.outerHeight = $button.outerHeight();
-
-    if (rtl) {
-      position.left -= MENU_WIDTH - $button.outerWidth();
-    }
+    const rtl = document.querySelectorAll("html.rtl")[0];
```

Comment:
```suggestion
    const rtl = document.documentElement.classList.contains("rtl");
```

Makes the intent clearer (i.e. `rtl` is a boolean again)